modin/core/execution/ray/implementations/pandas_on_ray/partitioning/partition_manager.py

```diff
@@ -111,6 +111,22 @@ def get_objects_from_partitions(cls, partitions):
         """
         return ray.get([partition._data for partition in partitions])
 
+    @classmethod
+    def wait_partitions(cls, partitions):
+        """
+        Wait on the objects wrapped by `partitions` in parallel, without materializing them.
+
+        This method will block until all computations in the list have completed.
+
+        Parameters
+        ----------
+        partitions : np.ndarray
+            NumPy array with ``PandasDataframePartition``-s.
+        """
+        ray.wait(
+            [partition._data for partition in partitions], num_returns=len(partitions)
```

Comment:
is there a reason to specify `num_returns` here?.. I think we can safely drop it

Reply:
If you don't, I believe ray will not wait for all objectrefs to be ready.

Reply:
Rehan is right, `num_returns` means the number of elements which should finish the execution and put result to object storage (https://docs.ray.io/en/latest/ray-core/package-ref.html#ray-wait)

Reply:
and the default value for the argument is 1, so we do need to specify it